### ISSUE TITLE
Add documentation for main page and relevant sub pages

### ISSUE THREAD

**Issue by holmeso** [AdamaJava/adamajava#15]
Need some doco.
High level from the adamajava page with details on what it is how to comple/run etc
Also need sub-project specific pages for some of the key components (eg. qprofiler) so that they can be built in isolation.
It would also be nice to add the teamcity widget that shows if the build is passing.

**Comment by delocalizer** [comment]
Addressing the last point, I added the commit status publisher feature to TC build:
http://bionode05.adqimr.ad.lan:8111/admin/editBuildFeatures.html?id=buildType:AdamaJava_GithubAdamajavaFullAutomaticBuildUpdatesCurrent
If I read the docs right, that should show build status in PRs

**Comment by delocalizer** [comment]
Seems like we also need a CONTRIBUTING.md

**Comment by holmeso** [comment]
Agree, although its not often you see github sites that ask people not to contribute...
